.drone.yml

```diff
@@ -117,9 +135,34 @@ pipeline:
       matrix:
         TEST_SUITE: integration
 
+  lint-test:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - find . -name \*.php -not -path './vendor/*' -exec php -l "{}" \;
```

Comment:
please use this tool instead - https://github.com/JakubOnderka/PHP-Parallel-Lint